src/Components/Post.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+const Posts = () => {
+  return (
+    <div className='Posts'>
+    <h1> Posts </h1>
+    <div className='PostCard'>
```

Comment:
This is not quite right, you would want to create a component that takes in props and uses those props to fill the information on it.